### ISSUE TITLE
Syntax Error in apps_helper

### ISSUE THREAD

**Issue by jgoetze** [errbit/errbit#1199]
Hi, I tried to update Errbit from 0.3.0 to 0.6.0. Doing that, I encounterd that error below. So I set it up from scratch. Its deployed via capistrano, running on a nginx. I still got the same error. The only post regarding this pointed to a wrong ruby version, but I checked again and I am running 2.1.0. what should be ok to run. 

I am confused, that it is a syntax error here. Anyone can help me with that?

```ruby
/srv/www/errbit/releases/20170524110905/app/helpers/apps_helper.rb:37: syntax error, unexpected keyword_do_block, expecting keyword_end
    apps.each do |app|
                ^
/srv/www/errbit/releases/20170524110905/app/helpers/apps_helper.rb:44: syntax error, unexpected keyword_end, expecting end-of-input (SyntaxError)
  /usr/local/rvm/gems/ruby-2.1.0@errbit/gems/activesupport-4.2.8/lib/active_support/dependencies.rb:274:in `require'
  /usr/local/rvm/gems/ruby-2.1.0@errbit/gems/activesupport-4.2.8/lib/active_support/dependencies.rb:274:in `block in require'
  /usr/local/rvm/gems/ruby-2.1.0@errbit/gems/activesupport-4.2.8/lib/active_support/dependencies.rb:240:in `load_dependency'
  /usr/local/rvm/gems/ruby-2.1.0@errbit/gems/activesupport-4.2.8/lib/active_support/dependencies.rb:274:in `require'
  /usr/local/rvm/gems/ruby-2.1.0@errbit/gems/activesupport-4.2.8/lib/active_support/dependencies.rb:360:in `require_or_load'
  /usr/local/rvm/gems/ruby-2.1.0@errbit/gems/activesupport-4.2.8/lib/active_support/dependencies.rb:317:in `depend_on'
  /usr/local/rvm/gems/ruby-2.1.0@errbit/gems/activesupport-4.2.8/lib/active_support/dependencies.rb:233:in
```

**Comment by stevecrozz** [comment]
Something doesn't seem right about your Ruby. Can you triple check the ruby version using RUBY_VERSION in a rails console or just irb?

Starting with version 2.1.0, Ruby allows an inline syntax for declaring private methods because defining a method now returns a symbol corresponding to the method name. You should get answers like these from the console:
```ruby
~/Projects/errbit/errbit (master *$=)$ ./bin/rails console
[1] pry(main)> RUBY_VERSION
=> "2.1.2"
[2] pry(main)> def foo; end
=> :foo
[3] pry(main)> private def bar; 'working private method'; end
=> Object
[4] pry(main)> bar
=> "working private method"
```

**Comment by jgoetze** [comment]
Thanks for your reply, stevecrozz.
I checked the version again and it worked. I have upgraded to 2.3.4 in the meantime.

It workes now. I have just tried to run the bootstrap task on the server. (after upgrading to 2.3.4 - so dont know if this is relevant) This now worked. Before, after creating the initial admin user it failed, but know he was able to create the indexes. Dont know, what was messed up. Maybe something wrong with the old database?
